syntax/vm/vm_stdlib_test.go

```diff
@@ -215,6 +215,30 @@ func TestStdlib_StringFunc(t *testing.T) {
 	}
 }
 
+func TestStdlibFileFunc(t *testing.T) {
+	tt := []struct {
+		name   string
+		input  string
+		expect interface{}
+	}{
+		{"file.path_join", `file.path_join("this/is", "a/path")`, "this/is/a/path"},
+		{"file.path_join empty", `file.path_join()`, ""},
```

Comment:
Is this going to work on Windows where paths use \ ?

Reply:
yeah it should, it uses the filepath.Join function which is using an OS specific separator (https://pkg.go.dev/path/filepath#Join)